contrib/snooze/snooze.c

```diff
@@ -27,7 +27,6 @@ pre_ckpt()
            dmtcp_get_generation());
 
   dmtcp_set_ckpt_dir(newCkptDir);
-  dmtcp_set_coord_ckpt_dir(newCkptDir);
```

Comment:
Do we not need to update the `ckpt_dir` info in the coordinator anymore?

Reply:
I agree with @JainTwinkle .  If we want to discuss removing some existing functionality, let's make that a separate PR, and not mix it with this one.